datalake/src/main/java/com/sequenceiq/datalake/flow/dr/backup/DatalakeBackupFlowConfig.java

```diff
@@ -75,10 +78,20 @@ public class DatalakeBackupFlowConfig extends AbstractFlowConfiguration<Datalake
                     .failureState(DATALAKE_BACKUP_FAILED_STATE)
                     .failureEvent(DATALAKE_BACKUP_FAILED_EVENT)
 
+                    .from(DATALAKE_FULL_BACKUP_IN_PROGRESS_STATE)
```

Comment:
What happens if the cancellation occurs during the DB backup? Will the DB backup complete normally and then go to the full backup then the cancellation state? Or will it be interrupted and result in the current condition?

Reply:
according to what I can read from the State Machine it will do exactly as you described - wait for DB operation complete - via Cloudbreak, Salt etc, not from Thunderhead where state is cancelled - then move to full backup and then it will recognize the state is cancelled - because this is where I believe the Flow will ask Thunderhead next time - and display "cancelled" state.